src/DependencyInjection/NelmioSolariumExtension.php

```diff
@@ -69,6 +69,8 @@ public function load(array $configs, ContainerBuilder $container)
             if ($name === $defaultClient) {
                 $container->setAlias('solarium.client', new Alias($clientName, true));
                 $container->setAlias($clientClass, new Alias($clientName, true));
+            } elseif (method_exists($container, 'registerAliasForArgument')) {
```

Comment:
I believe now since we bumped the minimum Symfony version to 4.4 this method probably always exists?